addons/account/views/res_config_settings_views.xml

```diff
@@ -149,7 +149,7 @@
                                     </div>
                                 </div>
                             </div>
-                            <div class="col-12 col-lg-6 o_setting_box" id="record_transactions">
+                            <div class="col-12 col-lg-6 o_setting_box" id="record_transactions" invisible="1">
```

Comment:
you can remove the whole block then, right?

Reply:
`<field name="group_multi_currency"/>` must be in the view in order to use  `attrs="{'invisible': [('group_multi_currency', '=', False)]}"` - the latter being used by a few other `res.config.settings` 

I can clean up the contents of the div though...?

Reply:
decided to move the `<field name="group_multi_currency" invisible="1"/>` somewhere else and removed the block